example_compute.py

```diff
@@ -29,7 +27,7 @@
 # <Node: provider=Rackspace, status=REBOOT, name=korine, ip=6.7.8.9.10>, ... ]
 
 # grab the node named "test"
-node = [n for n in nodes if n.name == 'test'][0]
+#node = [n for n in nodes if n.name == 'test'][0]
 
 # reboot "test"
-node.reboot()
+#node.reboot()
```

Comment:
Any reason for this to be commented out?

Reply:
Actually this file should not be commited, I will revert this one.